### ISSUE TITLE
[bug] failed to resolve host name

### ISSUE THREAD

**Issue by knassar702** [knassar702/scant3r#54]
this error is because of the multi-threading feature, so for now, you can set the delay option to 2 seconds for avoiding this error

```bash
$ cargo r -- urls --file urls.txt.1 --config config.yaml -c 100 --delay 2
```

**Comment by knassar702** [comment]
fixed 30cf622d9693460875ccd56460b4d52cce9b03a1

**Comment by knassar702** [comment]
it should be fixed now
also don't forget to change the requests limit on your machine
```
$ ulimit -n 3000
```
